### DISCUSSION TITLE
Folder name convention of Blade and Livewire components/layouts

### DISCUSSION THREAD

**Discussion by bilogic** [livewire/livewire#6057] [1. Help] [answered]
I'm trying to make sense of my resource folders

1. `/resources/views/components/` is used by Blade components, specifically, `<x-widget ...` translates to `/resources/views/components/widget.blade.php`. Creating Livewire components in this folder can make things confusing since we are never going to use `<x-livewire ...`, why not create them in something like `/resources/views/livewire/` instead?
2. On top of that, the default layout of a full page Livewire component is `/resources/views/components/layouts/app.blade.php`, I thought `/resources/views/layouts/app.blade.php` would make more sense, since they are just layouts. I don't find anything that different between a normal blade and livewire layout. (Or did I miss something?)
3. Bottomline, I find that allowing only Blade components in `/resources/views/components/` makes it easier for me to see what's what without having to open every file to examine them. 

Or did I miss something about the way it is currently structured? Thank you.

**Comment by joshhanley** [comment, marked as answer]
@bilogic  I'm not sure I follow, all Livewire views get stored in `resources/views/livewire`.

In regards to layouts, you can put the default layout wherever you like and just publish and update the Livewire config `livewire.layout` to point to your preferred layout location.

Hope this helps!

**Reply by bilogic** [reply]
yea thanks! I think I got confused about the livewire views
